### DISCUSSION TITLE
Welcome to go-tg Discussions!

### DISCUSSION THREAD

**Discussion by mr-linch** [mr-linch/go-tg#128] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mygoonet** [comment]
how make you own keyboard not inline,make pls example

**Reply by mygoonet** [reply]
when i use inlinekeyboard a cant get data from webapp telegram
